Invoro.WebApp/Invoro/invoro/src/lib/components/Status/Status.component.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { mount, shallow, ReactWrapper } from 'enzyme';
+import StatusComponent from './Status.component';
+import Status from '../../dataModel/Status';
+
+describe("StatusComponent", () => {
+    it("when status is NotPlanned, the label is corresponding", () => {
+        // Prepare
+        let props = {
+            status: Status.NotPlanned
+        };
+
+        // Action
+        let statusComponent: ReactWrapper = mount(<StatusComponent {...props} />);
+
+        // Assert
+        const chip = statusComponent.find("Chip");
```

Comment:
You can extract to method:

AssertIfChipStatusIs(statusStr: string)
{
       const chip = statusComponent.find("Chip");
        expect(chip.prop('label')).toBe(statusStr);
}